inc/admin/upgrader.php

```diff
@@ -95,8 +91,10 @@ function _imagify_new_upgrade( $imagify_version, $current_version ) {
 		}
 
 		// Auto-activate the Admin Bar option.
-		$options                   = get_site_option( IMAGIFY_SETTINGS_SLUG );
+		$options = get_site_option( IMAGIFY_SETTINGS_SLUG );
+		$options = is_array( $options ) ? $options : array();
```

Comment:
`$options = (array) $options;` directly?

Reply:
No, because if `$options` is `false`, doing `$options = (array) $options;` will result in `array( false )`.